custom_enumerable_methods.rb

```diff
@@ -0,0 +1,117 @@
+module Enumerable
+  def my_each
```

Comment:
#my_each, when a block is given and when it is a range, returns the range itself after calling the given block once for each element in self.
Consider the following example.
![Screen Shot 2020-07-30 at 7 45 34 AM](https://user-images.githubusercontent.com/52430951/88924455-ecbc8200-d238-11ea-85cc-501cca91571c.png)

Surrounding of color blue we have the expected result. 
In red the actual result.